lib/ditto.ts

```diff
@@ -94,6 +96,31 @@ const COMMANDS: CommandConfig<Command>[] = [
       },
     },
   },
+  {
+    name: "import-components",
+    description:
+      "Import components via a file. For more information please see: https://www.dittowords.com/docs/importing-string-files.",
+    flags: {
+      "-t, --text [value]": {
+        description: "Text column index",
```

Comment:
I wonder if it makes sense to include something like this for all of the flags here?
```suggestion
        description: "Text column index (.csv format only)",
```